src/colorella/conversions.py

```diff
@@ -177,7 +175,6 @@ def json2list(filepath):
             return name, colors, gradient
     else:
         raise ImportError("file ", filepath, "not found")
-        return None
```

Comment:
Could you please remove this also for the other file processes where an ImportError is raised?